frontend/src/pages/CreateProject.svelte

```diff
@@ -252,13 +252,18 @@ Co-founder at xxx`,
           recaptchaToken,
         }),
       })
-        .then((res) => {
+        .then(async (res) => {
           if (res.status === 200) {
             return res.json();
           } else if (res.status === 400) {
             console.log("Couldn't update project: ", res);
-            errorMessage =
-              "Error updating project. Please check fields. " + res;
+            res = await res.text();
+            try {
+              res = JSON.parse(res);
+            } catch (e) {}
+            errorMessage = `Error creating project. Please check fields. ${
+              res.message ?? res
```

Comment:
Great to see the error being visualized.

`res.message ?? res`
Am I reading this right? is this ternary?

in other words...
`if res.message !== undefined then return res else return null;`

Reply:
This is [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)